runtime/gc_modron_startup/mmparseXXgc.cpp

```diff
@@ -787,10 +787,7 @@ gcParseXXgcArguments(J9JavaVM *vm, char *optArg)
 			continue;
 		}
 		if(try_scan(&scan_start, "softwareEvacuateReadBarrier")) {
-			/* Software read barriers are only implemented on s390 for now */
-#if defined(S390) || defined(J9ZOS390)
 			extensions->softwareEvacuateReadBarrier = true;
-#endif /* defined(S390) || defined(J9ZOS390) */
```

Comment:
I'd rather be very specific and allow only for platforms we enable CS.

Reply:
This code has already be guarded by `OMR_GC_CONCURRENT_SCAVENGER`, which is only defined on platforms that we have enabled CS. IMO, another layer of "#ifdef" only introduces confusion but no actually benefits.

Reply:
make sense., I did not check this was already under OMR_GC_CONCURRENT_SCAVENGER